devtools/oil_gdb.py

```diff
@@ -152,10 +152,6 @@ def __call__(self, val):
       #print('target name %r' % target.name)
       #print('target tag %r' % target.tag)
 
-      if target.name == 'gc_heap::Str':
-          return GcStrPrinter(val)
-
-      # TODO: remove Str when we remove mylib_old.cc
```

Comment:
I think this should be changed to GcStrPrinter, or just revert this file ... we can take care of it when we hit it

Reply:
Updated